pkg/services/action_test.go

```diff
@@ -505,136 +438,12 @@ func (s *ServicesTestSuite) TestNewSubmitBizCase() {
 			sendRequesterEmail,
 			sendReviewerEmail,
 			submitToCEDARStub,
-			status,
 		)
 		err := submitBusinessCase(ctx, &intake, &action)
 
 		s.IsType(&apperrors.QueryError{}, err)
 	})
 
-	s.Run("does not return error for validation if status is not biz case final", func() {
```

Comment:
double check - was this ok to remove? or is this functionality still needed but just depends on different data?

Reply:
I _think_ we should probably keep the test, but instead of the test setting
```go
		status := models.SystemIntakeStatusBIZCASEDRAFTSUBMITTED
```

it should probably just set the intake's `step`

```go
		Step: models.SystemIntakeStepFINALBIZCASE

```